src/Super_Admin_Command.php

```diff
@@ -57,18 +57,17 @@ public function __construct() {
 	 *
 	 * @subcommand list
 	 */
-	public function _list( $_, $assoc_args ) {
+	public function list_( $_, $assoc_args ) {
```

Comment:
I'd go for something more meaningful here, as ending in an underscore is still "weird" in terms of method name.
```suggestion
	public function list_subcommand( $_, $assoc_args ) {
```